mcp/motors/ec_motors.c

```diff
@@ -58,16 +58,40 @@ static ph_thread_t *ecmonitor_ctl_id;
 
 extern int16_t InCharge;
 
+#define ECAT_SENDRECV_STACKSIZE (64 * 1024)
```

Comment:
would like a comment on this for future ref

Reply:
Setting the maximum allowed stack size for a RT thread is a complex topic. The default is something like 8 MB, according to Google, which is kind of a lot, and doesn't seem justified for these small functions. We might not use that much. I just used the same value as set in the example, 64 KB. I might go on a little detour to see how deep our stack actually gets in these functions to justify this choice. Either way, I'll leave a comment.

https://www.embeddedrelated.com/showarticle/1330.php

Reply:
I went with 8 MB because we have a lot of memory and we don't use a lot.